src/DependencyEmitter/UsesDependencyEmitter.php

```diff
@@ -34,4 +37,21 @@ public function applyDependencies(AstMap $astMap, Result $dependencyResult): voi
             }
         }
     }
+
+    protected function isNamespace(AstDependency $dependency, AstMap $astMap): bool
```

Comment:
Minor thing, but I think `isNamespace` does not really convey what is checked. Maybe something like `hasSharedNamespace()`?

Reply:
I was thinking of namespace as a path to a FQDN (i.e. `Qossmic\Deptrac\` is a namespace while `Qossmic\Deptrac\AstRunner\AstMap\AstDependency` is not a namespace it's a FQDN). With that understanding I thought `isNamespace` made sense. If you consider that a FQDN is also a namespace then it doesn't.

Maybe we can disambiguate by inverting the condition to `isFQDN`?